### PR TITLE
PRO-6099: Fix address forms bugs

### PR DESCRIPTION
**Pull request by LucaDelBuonoHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRO-6099

### Change description ###
Fix bug with address forms

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: app/resources/en/translation/applicant/address.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
   "errors": {
     "addressLine1": {
       "required": {
-        "summary": "Enter the first line of the address",
+        "summary": "You haven&rsquo;t entered the first line of the address",
```
Comment (douglasrice) [inline app/resources/en/translation/applicant/address.json]: Please revert this to the original text. 
From UX testing and GDS we should not use the pronoun "You" at the start of a sentence

File: app/resources/en/translation/applicant/address.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     },
     "postTown": {
       "required": {
-        "summary": "Enter a town or city",
+        "summary": "You haven&rsquo;t entered the town or city",
```
Comment (douglasrice) [inline app/resources/en/translation/applicant/address.json]: Please revert this to the original text. 
From UX testing and GDS we should not use the pronoun "You" at the start of a sentence

File: app/resources/en/translation/applicant/address.json [hunk 3]
```diff
@@ -35,12 +35,12 @@
     },
     "newPostCode": {
       "required": {
-        "summary": "Enter a postcode",
+        "summary": "You haven&rsquo;t entered the postcode",
```
Comment (douglasrice) [inline app/resources/en/translation/applicant/address.json]: Please revert this to the original text. 
From UX testing and GDS we should not use the pronoun "You" at the start of a sentence

File: app/resources/en/translation/applicant/address.json [hunk 4]
```diff
@@ -35,12 +35,12 @@
...
         "message": "Enter a postcode"
       },
       "invalid": {
-        "summary": "Please enter minimum length for post code",
-        "message": "2 - 14 characters"
+        "summary": "You haven&rsquo;t entered a complete postcode",
```
Comment (douglasrice) [inline app/resources/en/translation/applicant/address.json]: Please revert this to the original text. 
From UX testing and GDS we should not use the pronoun "You" at the start of a sentence

File: app/resources/en/translation/deceased/address.json [hunk 5]
```diff
@@ -34,12 +34,12 @@
...
     "newPostCode": {
       "required": {
-        "summary": "Enter a postcode",
+        "summary": "You haven&rsquo;t entered the postcode",
         "message": "Enter a postcode"
       },
       "invalid": {
-        "summary": "Please enter minimum length for post code",
-        "message": "2 - 14 characters"
+        "summary": "You haven&rsquo;t entered a complete postcode",
```
Comment (douglasrice) [inline app/resources/en/translation/deceased/address.json]: Please revert this to the original text. 
From UX testing and GDS we should not use the pronoun "You" at the start of a sentence

File: app/resources/en/translation/deceased/address.json [hunk 6]
```diff
@@ -34,12 +34,12 @@
     },
     "newPostCode": {
       "required": {
-        "summary": "Enter a postcode",
+        "summary": "You haven&rsquo;t entered the postcode",
```
Comment (douglasrice) [inline app/resources/en/translation/deceased/address.json]: Please revert this to the original text. 
From UX testing and GDS we should not use the pronoun "You" at the start of a sentence

File: app/resources/en/translation/deceased/address.json [hunk 7]
```diff
@@ -24,7 +24,7 @@
     },
     "postTown": {
       "required": {
-        "summary": "Enter a town or city",
+        "summary": "You haven&rsquo;t entered the town or city",
```
Comment (douglasrice) [inline app/resources/en/translation/deceased/address.json]: Please revert this to the original text. 
From UX testing and GDS we should not use the pronoun "You" at the start of a sentence

File: app/resources/en/translation/executors/address.json [hunk 8]
```diff
@@ -35,12 +35,12 @@
...
     "newPostCode": {
       "required": {
-        "summary": "Enter a postcode",
+        "summary": "You haven&rsquo;t entered the postcode",
         "message": "Enter a postcode"
       },
       "invalid": {
-        "summary": "Please enter minimum length for post code",
-        "message": "2 - 14 characters"
+        "summary": "You haven&rsquo;t entered a complete postcode",
```
Comment (douglasrice) [inline app/resources/en/translation/executors/address.json]: Please revert this to the original text. 
From UX testing and GDS we should not use the pronoun "You" at the start of a sentence

File: app/resources/en/translation/executors/address.json [hunk 9]
```diff
@@ -35,12 +35,12 @@
     },
     "newPostCode": {
       "required": {
-        "summary": "Enter a postcode",
+        "summary": "You haven&rsquo;t entered the postcode",
```
Comment (douglasrice) [inline app/resources/en/translation/executors/address.json]: Please revert this to the original text. 
From UX testing and GDS we should not use the pronoun "You" at the start of a sentence

File: app/resources/en/translation/executors/address.json [hunk 10]
```diff
@@ -25,7 +25,7 @@
     },
     "postTown": {
       "required": {
-        "summary": "Enter a town or city",
+        "summary": "You haven&rsquo;t entered the town or city",
```
Comment (douglasrice) [inline app/resources/en/translation/executors/address.json]: Please revert this to the original text. 
From UX testing and GDS we should not use the pronoun "You" at the start of a sentence

Comment (LucaDelBuonoHMCTS) [reply app/resources/en/translation/executors/address.json]: This is consistent with all other error messages we have across the app.

**Pull request merged**